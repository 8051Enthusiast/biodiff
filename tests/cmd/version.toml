bin.name = "biodiff"
args = ["--version"]
status.code = 0