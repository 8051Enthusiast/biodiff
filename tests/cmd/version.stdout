biodiff 1.2.0
